index.js

```diff
@@ -133,6 +133,32 @@ function wrap (argv, env, workingDir) {
       options.envPairs.push((isWindows ? 'Path=' : 'PATH=') + workingDir)
     }
 
+    if (file === 'npm') {
```

Comment:
You’re right – I’ll move the code for looking up `npmPath` into its own function. I’d like to keep the rest in place, if you don’t mind, because it resembles what happens in the blocks for handling `file === 'sh' || …` and `file === 'node' || …`.